### ISSUE TITLE
The default password for server_username is correct ?

### ISSUE THREAD

**Issue by devansvd** [opendistro/for-elasticsearch-docs#84]
https://opendistro.github.io/for-elasticsearch-docs/docs/security/multi-tenancy/

Running in 0.9.0 both odfe-elasticsearch and odfe-kibana.

Are you sure the default `server_username` for multi tenancy is `kibanaserver` ? It throws 403 & I found the below error in the logs. But changing the `server_username` to `admin` works perfectly.

![odfe](https://user-images.githubusercontent.com/11324776/58746893-6c0dc480-8481-11e9-8543-8861881185ec.png)

`{"type":"error","@timestamp":"2019-06-01T07:35:37Z","tags":["error","migration"],"pid":1,"level":"error","error":{"message":"Authorization Exception: Authorization Exception","name":"Error","stack":"Authorization Exception :: {\"path\":\"/_opendistro/_security/tenantinfo\",\"query\":{},\"statusCode\":403,\"response\":\"\"}\n    at respond (/usr/share/kibana/node_modules/elasticsearch/src/lib/transport.js:308:15)\n    at checkRespForFailure (/usr/share/kibana/node_modules/elasticsearch/src/lib/transport.js:267:7)\n    at HttpConnector.<anonymous> (/usr/share/kibana/node_modules/elasticsearch/src/lib/connectors/http.js:166:7)\n    at IncomingMessage.wrapper (/usr/share/kibana/node_modules/elasticsearch/node_modules/lodash/lodash.js:4935:19)\n    at IncomingMessage.emit (events.js:194:15)\n    at endReadableNT (_stream_readable.js:1103:12)\n    at process._tickCallback (internal/process/next_tick.js:63:19)"},"message":"Authorization Exception: Authorization Exception"}
`

**Edit**
I'm running 0.9.0 wrongly mentioned 0.7.0

**Comment by aetter** [comment]
Hey @devansvd, a few things:

- Can you retry on a newer version of ODFE? 0.7.0 has a bug that makes multi-tenancy basically unusable.
- Can you check `kibana.yml` and `plugins/opendistro_security/securityconfig/internal_users.yml` to make sure that you're using the same user/password everywhere? If you changed the password in `internal_users.yml`, you need to specify the new one in Kibana.
- Can you veirfy that you've set `elasticsearch.requestHeadersWhitelist: ["securitytenant","Authorization"]` in `kibana.yml`?

**Comment by devansvd** [comment]
- I'm using the 0.9.0 version, not 0.7.0(I wrongly mentioned 0.7.0)
- I haven't touched internal_users.yml yet
- yes,  i have whitelisted the request headers. The default password kibanaserver doesn't work for me. 

I was able to resolve the issue by changing server_name to admin.

**Comment by aetter** [comment]
Hi @devansvd, changing that setting to `admin` shouldn't be necessary and isn't recommended. The `kibanaserver` user receives a very specific set of permissions necessary to operate Kibana, whereas the `admin` user has full permissions.

My advice here is the still to verify that `config.yml` and `kibana.yml` both specify the same user and/or password from `internal_users.yml` and then [load the configuration into the cluster](https://opendistro.github.io/for-elasticsearch-docs/docs/security/security-admin/) using `securityadmin.sh`.

If you post `kibana.yml`, `config.yml`, and `elasticsearch.yml` here, I can dig a little more. Are you using the Docker, RPM, or APT install?

**Comment by devansvd** [comment]
@aetter Indeed I'm was using the same elasticsearch username - `kibanaserver` in both config.yml & kibana.yml. Still I get the Authorization Exception error. What you said `kibanaserver` user has to be limited access to operate only kibana whereas `admin` gives has full permission makes much sense & sounds right.

The why the exception occurs ? I found while starting docker image for kibana I passed few env -e ELASTICSEARCH_USERNAME=admin -e ELASTICSEARCH_PASSWORD=admin this overrides the kibana.yml user & password setting.

Removing this env's while starting docker kibana solved the issue. Thanks and much appreciate for the help and thoughts.
